From 6996bbd89d001c6071eedf3ddf6245c12bf9d92d Mon Sep 17 00:00:00 2001
From: "Daniel T. Borelli" <daltomi@aol.com>
Date: Mon, 15 Aug 2016 11:40:48 -0300
Subject: [PATCH] Clarify calculation precedence for & and ?

---
 trunk/protocol.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/trunk/protocol.c b/trunk/protocol.c
index ca077e8..ebc0f15 100644
--- a/trunk/protocol.c
+++ b/trunk/protocol.c
@@ -293,7 +293,7 @@ void packet_buf_add_tags (struct packet_buf *b, const struct file_tags *tags)
 		packet_buf_add_str (b, tags->artist ? tags->artist : "");
 		packet_buf_add_str (b, tags->album ? tags->album : "");
 		packet_buf_add_int (b, tags->track);
-		packet_buf_add_int (b, tags->filled & TAGS_TIME ? tags->time : -1);
+		packet_buf_add_int (b, (tags->filled & TAGS_TIME) ? tags->time : -1);
 		packet_buf_add_int (b, tags->filled);
 	}
 	else {
-- 
2.9.3

