From 07292fee099b622bbb5298ee702686e20e122d27 Mon Sep 17 00:00:00 2001
From: "Daniel T. Borelli" <daltomi@openmailbox.org>
Date: Sat, 3 Jun 2017 13:36:26 -0300
Subject: [PATCH] Invalid wcstombs() argument nr 3.

---
 trunk/interface_elements.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/trunk/interface_elements.c b/trunk/interface_elements.c
index 0e83af9..345d85f 100644
--- a/trunk/interface_elements.c
+++ b/trunk/interface_elements.c
@@ -367,7 +367,7 @@ static void entry_draw (const struct entry *e, WINDOW *w, const int posx,
 			sizeof(wchar_t) * (len + 1));
 	if (len > e->width)
 		text_ucs[e->width] = L'\0';
-	len = wcstombs (NULL, text_ucs, -1) + 1;
+	len = wcstombs (NULL, text_ucs, 0) + 1;
 	assert (len >= 1);
 
 	text = (char *)xmalloc (len);
@@ -642,7 +642,7 @@ static char *entry_get_text (const struct entry *e)
 
 	assert (e != NULL);
 
-	len = wcstombs (NULL, e->text_ucs, -1) + 1;
+	len = wcstombs (NULL, e->text_ucs, 0) + 1;
 	assert (len >= 1);
 	text = (char *) xmalloc (sizeof (char) * len);
 	wcstombs (text, e->text_ucs, len);
-- 
2.13.0

