From 86f8025a0aa58ae4aff53e39fe3ff34a9e28575a Mon Sep 17 00:00:00 2001
From: "Daniel T. Borelli" <daltomi@aol.com>
Date: Tue, 28 Jun 2016 02:07:04 -0300
Subject: [PATCH 3/5] Fix segfault.

- Remove second call poptGetOptArg.
---
 trunk/main.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/trunk/main.c b/trunk/main.c
index 51d0f3d..675afb8 100644
--- a/trunk/main.c
+++ b/trunk/main.c
@@ -1039,7 +1039,6 @@ static void process_options (poptContext ctx, lists_t_strs *deferred)
 			options_ignore_config ("ASCIILines");
 			break;
 		case CL_JUMP:
-			arg = poptGetOptArg (ctx);
 			params.jump_to = get_num_param (arg, &jump_type);
 			if (*jump_type)
 				if (!jump_type[1])
-- 
2.9.0

