From 3b8c72386105a6a20a20bd97f3439b0c2ad32979 Mon Sep 17 00:00:00 2001
From: "Daniel T. Borelli" <daltomi@aol.com>
Date: Sat, 2 Jul 2016 11:50:27 -0300
Subject: [PATCH 09/19] exp(x) - 1 replaced by expm1()

---
 trunk/jack.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/trunk/jack.c b/trunk/jack.c
index 645ee98..434e248 100644
--- a/trunk/jack.c
+++ b/trunk/jack.c
@@ -304,7 +304,7 @@ static int moc_jack_read_mixer ()
 static void moc_jack_set_mixer (int vol)
 {
 	volume_integer = vol;
-	volume = (jack_default_audio_sample_t)((exp((double)vol / 100.0) - 1)
+	volume = (jack_default_audio_sample_t)(expm1((double)vol / 100.0)
 			/ (M_E - 1));
 }
 
-- 
2.13.1

