From 19eacc238ee85a09a258fc0999eb4cd9b8bcb486 Mon Sep 17 00:00:00 2001
From: "Daniel T. Borelli" <daltomi@openmailbox.org>
Date: Sun, 4 Jun 2017 10:50:51 -0300
Subject: [PATCH 13/18] Replace sprintf -> asprintf

---
 trunk/server.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/trunk/server.c b/trunk/server.c
index 6a813ee..3f6be6d 100644
--- a/trunk/server.c
+++ b/trunk/server.c
@@ -1339,13 +1339,15 @@ void req_equalizer_next()
 
 void req_toggle_make_mono()
 {
-	char buffer[128];
+	char * buffer = NULL;
 
 	softmixer_set_mono(!softmixer_is_mono());
 
-	sprintf(buffer, "Mono-Mixing set to: %s", softmixer_is_mono()?"on":"off");
+	asprintf(& buffer, "Mono-Mixing set to: %s", softmixer_is_mono()?"on":"off");
 
 	status_msg(buffer);
+
+	free (buffer);
 }
 
 /* Handle CMD_GET_FILE_TAGS. Return 0 on error. */
-- 
2.13.0

